app/controllers/api/schedules_controller.rb

```diff
@@ -42,5 +42,14 @@ def sequence
     def schedule
       @schedule ||= Schedule.find(params[:id])
     end
+
+    def default_serializer_options
+      # For some strange reason, angular-data crashes if we don't call super()
+      # here. Rails doesn't care, though.
+      super.merge({
+        start: params[:start],
+        finish: params[:finish]
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.